idsvr/README.md

```diff
@@ -42,78 +42,80 @@ Tutorials on using the chart and more are available in the [Curity Resource libr
-`curity.config.configurationConfigMap`| *DEPRECATED* The ConfigMap containing configuration which is mounted as a volume                                                     | `null`
-`curity.config.configurationConfigMapItemName`| *DEPRECATED* The `curity.config.configurationConfigMap`'s item name, required if the ConfigMap is set.                               | `null`
-`curity.config.configuration` | The array of Configmaps and secrets configuration items are mounted as a volume                                                      | `[]`
-`curity.config.convertKeystore` | The array of secrets containing tls certificates that will be converted to Curity format                                             | `[]`
-`curity.config.backup` | If `true`, the configuration will be backed up in a secret in each commit                                                            | `false`
-`ingress.enabled`| Flag to enable/disable an Ingress resource                                                                                           | `false`
-`ingress.annotations`| Extra annotations for the Ingress resource                                                                                           | `{}`
-`ingress.runtime.host`| Hostname of the runtime servers (used by the Ingress resource)                                                                       | `curity.local`
-`ingress.runtime.paths`| Paths of the runtime servers that can be accessed externally                                                                         | `{/}`<sup>[2](#f2)</sup>
-`ingress.runtime.secretName`| Secret which contains the tls cert and key for the runtime TLS connection. If not set, the Ingress will be configured for HTTP       | `null`
-`ingress.admin.host`| Hostname for the admin server (used by the Ingress resource)                                                                         | `curity-admin.local`
-`ingress.admin.secretName`| Secret which contains the tls cert and key for the runtime TLS connection. If not set, the Ingress resource will be configured for HTTP | `null`
-`resources`| Resource limits applied in admin and runtime deployments                                                                             | `{}`
-`nodeSelector`| Node selector applied in admin and runtime deployments                                                                               | `{}`
-`tolerations`| Tolerations applied in admin and runtime deployments                                                                                 | `{}`
-`affinity`| Affinity applied in admin and runtime deployments                                                                                    | `{}`
+| Parameter                                           | Description                                                                                                                                   | Default                          |
```

Comment:
Same as with your other MR, you don't want `git blame` to show you as the owner for this entire table. Try to remove the whitespace changes so that only your single new row shows up in the diff.

Reply:
If I don't format the table, it was unclear to me what the spacing standard of this table is.  Different rows had different numbers of spaces, but the pipe characters didn't line up.

Reply:
I have changed the formatting on this table on master. You can use single spaces between `|` now to be consistent

Reply:
Fixed in a separate commit